lib/upgrade_js.js

```diff
@@ -123,8 +125,8 @@ class Script {
 
     // hostAttributes
     var hostAttrsProperties = [];
-    for (var key in hostAttrs) {
-      var astVal = {type: 'Literal', value: hostAttrs[key]};
+    for (let key in hostAttrs) {
```

Comment:
hostAttrs isn't an iterator though. I get the error `undefined is not a function` with this, I believe because it's looking up Symbol('@@iterator') on hostAttrs.